.travis.yml

```diff
@@ -19,6 +19,12 @@ addons:
     packages:
     - libssl-dev
 
+before_install:
```

Comment:
I prefer to do this at the end of the normal build, since I see this check as being of lesser importance, and doing it here halts the build early.